packages/turbo-utils/package.json

```diff
@@ -19,15 +19,17 @@
   "scripts": {
     "build": "tsup",
     "test": "jest",
-    "lint": "eslint src/**/*.ts"
+    "lint": "eslint src/**/*.ts",
+    "check-types": "tsc --noEmit"
   },
   "devDependencies": {
     "@manypkg/find-root": "^1.1.0",
     "@types/jest": "^27.4.0",
     "@types/node": "^16.11.12",
-    "esbuild": "^0.14.38",
+    "eslint": "^8.23.0",
```

Comment:
I personally like how the diff resolves to look like you replaced esbuild with eslint.

Reply:
Lol, this was in an effort to fix the `skipLibCheck` issue that you also pointed out. But it doesn't matter since it comes in via `tsup` anyway.